.changeset/moody-students-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+fix <Route> received an unexpected slot "default"
```

Comment:
```suggestion
fix `<Route> received an unexpected slot "default"` warning
```

We probably want to wrap this with backticks. I don't think the changelog would render `<Route>` correctly without backticks.